### PR TITLE
Wrap attributes in [] for angular binding where possible (Issue-3323)

### PR DESCRIPTION
**Pull request by DailisLangovskis**
## Description

Provide angular data binding to HTML attributes using [] brackets, due to having mixed styles, and brackets being used for the most part.

## Related issues or pull requests

Issue #3323 

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [ ] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [ ] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [ ] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by jmacura** [comment]
image

**Review by jmacura — APPROVED** [review]

File: projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html [hunk 1]
```diff
@@ -2,26 +2,30 @@
...
+                data.app} }}
                 <br>{{'COMPOSITIONS.dialogoverwriteConfirm.whatDoYouWant' | translateHs: {app: data.app} }}
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-primary compositions-btn-overwrite" (click)="overwrite()"
                     data-dismiss="modal">{{'COMMON.continue' | translateHs: {app: data.app} }}</button>
                 <button type="button" class="btn btn-primary compositions-btn-add" (click)="add();"
-                    data-dismiss="modal">{{'COMPOSITIONS.dialogoverwriteConfirm.mergeCompositions' | translateHs: {app: data.app} 
+                    data-dismiss="modal">{{'COMPOSITIONS.dialogoverwriteConfirm.mergeCompositions' | translateHs: {app:
+                    data.app}
```
Comment (jmacura) [inline projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: Prettier formatter?

Comment (DailisLangovskis) [reply projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: @jmacura Yes, a have set to format after save, just didn't expect so many occurrences.

Comment (jmacura) [reply projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: That's fine. I just dislike the way it wraps elements in HTML templates. But that's more my problem than yours 😄

Comment (DailisLangovskis) [reply projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: Sry, next files updates will come with ESLint 😄.

Comment (jmacura) [reply projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: Not sure if ESLint is capable of formatting HTML... I generally use VS Code's default HTML formatter or intend manually. That is a bad habit, of course...

Comment (DailisLangovskis) [reply projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html:24]: Yes, You are right! I will just set default formatter to VS Code's for HTML files.

**Pull request merged**